source/camera/CameraManager.h

```diff
@@ -24,6 +24,8 @@
 #define CAMERAMANAGER_H_
 
 #include "camera/HermiteCatmullSpline.h"
+#include "gamemap/GameMap.h"
+#include "entities/Tile.h"
```

Comment:
You should avoid to include headers as much as possible in other headers. Especially gamemap and tile which are pretty much used (and likely to be changed). That increases the compilation time.